Makefile

```diff
@@ -88,15 +89,22 @@ pypi: wheel source-pypi develop
 	@echo
 
 clean:
+	for PLUGIN in $(AVOCADO_OPTIONAL_PLUGINS); do\
+		if test -f $$PLUGIN/setup.py; then cd $$PLUGIN; $(PYTHON) setup.py clean --all; cd -; echo ">> UNLINK $$PLUGIN";\
+		else echo ">> SKIP $$PLUGIN"; fi;\
+	done
 	$(PYTHON) setup.py clean
 	rm -rf build/ MANIFEST BUILD BUILDROOT SPECS RPMS SRPMS SOURCES PYPI_UPLOAD
 	rm -f man/avocado.1
 	rm -rf docs/build
 	find docs/source/api/ -name '*.rst' -delete
+	$(PYTHON) setup.py clean --all
```

Comment:
Should the `--all` be added to the previous occurrence (Line 96)? I don't see a reason to call it twice.

Reply:
Yes, I thought about this too. Adding `clean --all` to this repo so would clean in a recursive way. And opt to make this change in future PRs. The clean command that I'm using (from distutils) is recommended, afaict. And I could not find a  straight forward solution for adding arguments, so far.

So, yes, I agree with you, but if you don't mind I would like to propose doing this in the future.